code/modules/reagents/chemistry/reagents/misc_reagents.dm

```diff
@@ -178,33 +178,33 @@
 	if(exposed_temperature <= T0C + 600)
 		return
 
-	var/violent = volume > 30
-	var/message_type = violent ? "class='boldwarning'" : "class='warning'"
-	holder.my_atom.visible_message("<span [message_type]>The oil [violent ? "boils and burns violently" : "burns"]!</span>")
+	// Let's say burning boiling oil doubles in volume, can our max volume contain that?
+	var/boil_overflow = (holder.total_volume + volume) - holder.maximum_volume
+	var/violent = boil_overflow > 0
```

Comment:
I don't think you need this variable anymore, just make the boil_overflow > 0 check in the if statement.

Reply:
Done, apologies for the wait; I completely forgot about this PR, lmao